lib/stack_master.rb

```diff
@@ -8,7 +8,9 @@
 require 'aws-sdk-sns'
 require 'aws-sdk-ssm'
 require 'colorize'
-require 'active_support/core_ext/string'
+require 'active_support/core_ext/hash/keys'
+require 'active_support/core_ext/object/blank'
+require 'active_support/core_ext/string/inflections'
```

Comment:
Previously, when requiring `active_support/core_ext/string` it'd load all these string extensions:

https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/string.rb

Which would in turn load a bunch of other core extensions.

Let's be a little more explicit about the extensions StackMaster uses, and only load these.